x-pack/plugins/screenshotting/server/cloud/read_cgroup_mem_limit.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Please note: the current implementation using cgroups will be changed to using the new cloud-provided environment variable that gives us the instance size:

https://github.com/elastic/cloud-assets/pull/1019